packages/renderer/src/lib/welcome/WelcomePage.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+import bgImage from './background.png';
+import DesktopIcon from '../images/DesktopIcon.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+</script>
+
+<div
+  class="flex flex-col flex-auto min-w-full bg-zinc-700 bg-no-repeat bg-top overflow-y-scroll"
+  style="background-image: url({bgImage})">
```

Comment:
@deboer-tim this looks amazing!! I think I only have one little aesthetic nit, and that's with the placement of the fuzzie circle gradient background... If it was possible to have it top-aligned vertically, and then pulled back -50% from the top of the screen, it would look more like the mockups. The way to do in plain css for a bg image I think would be adding something like this in the style attribute:

```
  background-position: 50% 200%;
  background-size: cover;
```

Here's a quick codepen:
https://codepen.io/mairindubh/pen/yLxgKQN

I don't know how to do in tailwind tho... maybe .bg-top then margin-top: -50% or something like that?

Reply:
Sorry, I was having problems with the background, and whenever I tried to use 'cover' or set the vertical size near 100% the position wouldn't be respected. I've played with it a bit and updated the screenshot above, let me know what you think.

Reply:
Looks perfect!!